### PR TITLE
Add optional creation of Bitbucket tasks for violation comments.

### PR DESCRIPTION
**Pull request by Syquel**
This feature optionally adds a task in Bitbucket to every violation comment, which is created on the pull request.
In combination with the Bitbucket merge check "No incomplete tasks" merges of pull requests are blocked as long as the violations are not resolved.

### CODE REVIEW

File: src/main/java/se/bjurr/violations/comments/bitbucketserver/lib/BitbucketServerCommentsProvider.java [hunk 1]
```diff
@@ -88,7 +88,12 @@ public void createCommentWithAllSingleFileComments(final String comment) {
   @Override
   public void createSingleFileComment(
       final ChangedFile file, final Integer line, final String comment) {
-    client.pullRequestComment(file.getFilename(), line, comment);
+    final BitbucketServerComment bitbucketComment =
+        client.pullRequestComment(file.getFilename(), line, comment);
+
+    if (violationCommentsToBitbucketApi.getCreateSingleFileCommentsTasks()) {
```
Comment (tomasbjerre) [inline src/main/java/se/bjurr/violations/comments/bitbucketserver/lib/BitbucketServerCommentsProvider.java]: You are changing `client.pullRequestComment` to return the comment and not actually create the comment.

If this is `false` then there will be no comment created at all right? Commenting single files, without creating a task, no longer works with this change?

Comment (tomasbjerre) [reply src/main/java/se/bjurr/violations/comments/bitbucketserver/lib/BitbucketServerCommentsProvider.java]: You are right! Merging it now.

**Comment by tomasbjerre** [comment]
I'm releasing the Jenkins plugin now, version **1.74**!

**Comment by Syquel** [comment]
Thanks for the quick release! 

I already integrated this option into the violation-comments-to-stash plugin, but unfortunately it's on my laptop at work. 
I will create a pull request on monday for it.

**Comment by tomasbjerre** [comment]
Ok. I added an option for it and released it.

**Pull request merged**